src/modules/settings/components/settings/AutoJoinRaids.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Panel from 'rsuite/lib/Panel/index.js';
+import Toggle from 'rsuite/lib/Toggle/index.js';
+import {registerComponent, useStorageState} from '../Store.jsx';
+import {SettingIds, CategoryTypes} from '../../../../constants.js';
+import styles from '../../styles/header.module.css';
+
+function AutoJoinRaids() {
+  const [value, setValue] = useStorageState(SettingIds.AUTO_JOIN_RAIDS);
+
+  return (
+    <Panel header="Auto Join Raids">
+      <div className={styles.toggle}>
+        <p className={styles.description}>Untoggle to be opted out of raids.</p>
```

Comment:
Can we update this to describe the enabled state rather than the disabled state?

Reply:
```suggestion
        <p className={styles.description}>Joins channel raids automatically without user interaction </p>
```
Unless I misunderstand something, this text edit is the last item blocking this merge. Is that correct?

Reply:
@eldorel Thanks for the reminder. I have updated the description, removing the "without user interaction" ending as it seemed redundant with "automatically".